scholia/app/templates/dataset.html

```diff
@@ -23,13 +23,14 @@ <h1 id="h1">Taxon</h1>
 
 <div id="intro"></div>
 
+<div id="details"></div>
```

Comment:
I think it goes after the wembedder div.

Reply:
got it

Reply:
let me modify it